fixtures/doctor.csv

```diff
@@ -0,0 +1,51 @@
+doctor_id,job,age
```

Comment:
I think we should provide the script that generates those data, WDYT?

Reply:
I don't think that's the role of this repo. I know it's a custom fixture, but we didn't provide anything from the other (ex: adult_with_cities). But I agree that it's interesting to have the code somewhere if we need to generate a wider or more complex use case.
I put this in Morgan analysis for now